packages/@okta/vuepress-site/docs/reference/architecture-center/directory-coexistence/overview/index.md

```diff
@@ -0,0 +1,103 @@
+* Merging different identity stores into a single identity provider after merging with or acquiring another company.
+
+### What Okta technologies are used?
+
+* Okta Universal Directory
+* Okta AD Agent
+* Okta LDAP Agent
+* Okta System for Cross-domain Identity Management (SCIM) connector
+* Okta Provisioning Agent
+
+## Architecture
+
+Before applying the directory coexistence architecture, a network's IAM solution is split across many identity providers (IdPs):
+
+* Cloud-based IdPs such as Microsoft Azure Active Directory
+* On-prem IdPs based on LDAP
+* On-prem IdPs based on a proprietary database
+
+Users may need several usernames and passwords to access their applications.
+
+<div class="full">
+
+  ![An architecture diagram representing the start state of the lab](/img/architecture/directory-coexistence/overview-lab-start-state.png)
+
+  <!--
+    Source image: fill-this-in overview-lab-start-state
+  -->
+</div>
+
+By applying the architecture for the user migration process:
+
+* All applications start the authentication flow by querying Universal Directory.
+* If the user has been migrated and their credentials are valid, they are signed in.
+* If the user has not been migrated, their authentication is delegated to the original identity provider for that application.
+
+From the user's perspective, nothing changes except for the number of different passwords to remember, which will reduce to one.
+
+<div class="full">
+
+  ![An architecture diagram representing the end state of the lab.](/img/architecture/directory-coexistence/overview-lab-end-state.png)
+
+  <!--
+    Source image: fill-this-in overview-lab-end-state
+  -->
+</div>
+
+Over time, as all user information is migrated, the delegation to each original identity provider can be stopped until, eventually, the migration is complete.
+
+## Key considerations
+
+Several questions should be considered before applying this architecture:
+
+1. Can you migrate users from a provider over time, or is it an all-in-one process?
+2. How do you find and merge duplicate user profiles?
+3. Can you update your applications to use OpenID Connect (OIDC)?
+
+By considering these, you can also start understanding how long it may take to apply this architecture to your network.
+
+### Just-in-time Migration
+
+Until you are ready to use only Okta for authentication, your users can sign into Okta, and Okta will delegate authentication to your current identity provider. Your current identity directories and Universal Directory may coexist as long as required.
+
+There are three strategies for user migration:
+
+1. In a **bulk migration** strategy, you divide users, groups, and data into sections. Create an order in which each section will be migrated as a whole, and a schedule for migrating each section. For example, you may migrate everything in one go or create twelve portions and migrate one a month.
```

Comment:
```suggestion
1. In a bulk migration strategy, you divide users, groups, and data into sections. Create an order in which each section will be migrated as a whole, and a schedule for migrating each section. For example, you may migrate everything in one go or create twelve portions and migrate one a month.
```

Reply:
@brianduffield-okta Why unbold this? This is exaclty what bolding is for...